docs/application/native/guides/app-management/tidl.md

```diff
@@ -19,9 +19,13 @@ Usage:
   -o, --output=OUTPUT         The generated interface file.
   -p, --proxy                 Generate proxy code.
   -s, --stub                  Generate stub code.
+  -r, --rpclib                Generate C# library for rpc-port.
+  -t, --thread                Generate thread code with stub code.
+  -b, --beta                  Use beta version for private file sharing.
   -v, --version               Show version information.
```

Comment:
Shows version information.

Reply:
Hi @hjhun, gentle reminder to fix the comments.

Reply:
Hi @annie-samsung, we don't usually use the third person singular in usage comments of the tool.
Please check help messages of other tools.